en/engine/event/event-emit.md

```diff
@@ -1,31 +1,39 @@
-# Listening to and launching events
+# Event Listening and Emitting
 
-## Listening to events
+Cocos Creator provides the `EventTarget` class to suport the listening and emitting of custom event. Before using it, it needs to be imported from the `'cc'` module and an `EventTarget` object needs to be instantiated.
+```ts
+import { EventTarget } from 'cc';
+const eventTarget = new EventTarget();
+```
+
+> __Note__: Although the `Node` object also implements some `EventTarget` interfaces, we no longer recommend continuing to use the `Node` object to listen to and emit custom events. Because this is not efficient enough, and we also hope that `Node` The object only cares about events related to `Node`.
```

Comment:
```suggestion
> __Note__: although the `Node` object also implements some `EventTarget` interfaces, it is no longer recommended to continue using the `Node` object to listen to and emit custom events. Because this is not efficient enough, and we also hope that `Node` The object only cares about events related to `Node`.
```

Reply:
But this statement:
```
Because this is not efficient enough, and we also hope that `Node` The object only cares about events related to `Node`.
```

doesn't make much sense to me, can you explain what you meant?

Reply:
Sorry, I updated this sentence https://github.com/cocos-creator/creator-docs/pull/1903/commits/ff275529711a93dc93ff7d78fe457106804eee07


What I want to emphasize is that we don't hope developers to coding like
```
this.node.on('foobar', callback, target);  // the event 'foobar' is not node-related

eventTarget.on('foobar', callback, target);  // we suggest to register 'foobar' on eventTarget, instead of node
```